asset/js/setup.js

```diff
@@ -33,6 +33,14 @@ function setupAnchors() {
   });
 }
 
+function getSearchData(vm) {
```

Comment:
Rename `getSearchData()` to `updateSearchData()` (methods with names that start with `get` are usually accessor methods).